flycheck-ert.el

```diff
@@ -357,6 +357,10 @@ check that the buffer has all ERRORS, and no other errors."
   (should (= (length errors)
              (length (flycheck-overlays-in (point-min) (point-max))))))
 
+(put 'flycheck-ert-suspicious-checker 'error-message
+     "Suspicious state from checker.")
+(put 'flycheck-ert-suspicious-checker 'error-conditions '(error))
```

Comment:
I just copied the way it was done for `flycheck-ert-syntax-check-timed-out` and `flycheck-ert-skipped` higher in the same file.  But it looks like `define-error` is a more direct way.  Want me to refactor the others using `define-error` as well?